### DISCUSSION TITLE
Section Labels

### DISCUSSION THREAD

**Discussion by TheSoberJewel** [CAWSCIT/ca-meeting-finder-issue-tracker#26] [Ideas]
When the user clicks on a meeting link, they are taken to the meeting specific page.  If that meeting happens to be at a physical location where several meetings take place, there is a section on the page that shows the Day/Time/etc for those other meetings happening at the same physical location.
1. Please consider adding a Section Label to the top of this area so the user understands why these other meetings are listed here.  Suggestion would be 'Additional Meetings in Same Location'  It really threw me off the first few times I saw this on the page as I didn't understand why these other meetings were appearing on the page.
2. Wish List idea - Could there be a way to collapse/expand this section within the page? I am an addict with zero patience some days.  I don't want to scroll past info I didn't ask for just to be able to get to the map.

Also on that meeting specific page, there is another section providing info about other meetings with the same name.
1. Please consider adding a Section Label to the top of this area so the user understands why this additional info is showing up.  Again, it threw me off the first few times I saw this info.  Suggestion would be 'Meetings with the same name.'
2. I find it very confusing when this section pulls in meetings by the same name but that are in completely unrelated geographic or virtual locations.  An example is A Vision For You (Tuesdays, 10P ET, Online).  The app is currently pulling in info for a meeting in Texas just because it has the same name.  This issue is just going to increase as more areas add their meetings to the app as this is a fairly common meeting name.  I understand what the feature is supposed to do. It is supposed to help people find info about the same group that holds multiple meetings a week.  However, it is just confusing when it pulls in info for unrelated meetings.
3. Wish List idea -- Could there be a way to collapse/expand this section within the page?  Again, I find it a pain-point to have to scroll past info I didn't ask for just to get to the map.

**Comment by enderv** [comment]
for grouping same name meetings filed https://github.com/CAWSCIT/ca-meeting-finder-issue-tracker/issues/29 which should hopefully fix that.

For the rest of the requests would be upstream to tsml-ui, we can try to follow up with them see what they say

**Comment by KalobTaulien** [comment]
The old TSML UI has been deprecated. We're using an in-house version of the Meeting Finder Interface with the old filter function being a backup while the new version gets released into the wild. 

These problems will be solved moving forward, and we now have full control over the Meeting Detail Screen, so we can add/remove data as needed.
